backend/app/app/config/tools.yml

```diff
@@ -24,6 +24,8 @@ library:
         content: |-
           Example Input: \"User question: Could you find a list of tracks that Daan Peeters has purchased?\"
           Example Output: \"Daan Peters has purchased a total of X tracks, please see a list below...\"
+    additional:
+      human_message_only: false
```

Comment:
Note this is included only for clarity - removing the line will not chaneg the behavior